.github/workflows/stale.yml

```diff
@@ -42,14 +42,21 @@ jobs:
         with:
           debug-only: ${{ inputs.dryRun || false }}
           operations-per-run: ${{ inputs.operationsPerRun || 100 }}
+          ascending: true
           days-before-stale: 90
-          days-before-close: -1
+          days-before-close: 120
```

Comment:
I know it's probably not doable, but it would be great to close only the PRs where the contributor dropped the ball, and leave the ones waiting for us maintainers to review open.
With the closing message we leave the door open for the contributor to reopen, maybe we can be more explicit stating that it's totally fine to reopen if the PR is under this case

Reply:
From the actions/stale docs

> It means that any updates made, or any comments added to the issues or to the pull requests will restart the counter of days before marking as stale.

So, if the contributor adds a comment, it will reset the counter for being stale. I'll clarify this in the stale message.

There is probably some additional automation we can look into for raising our attention to do reviews.